app/helpers/repositories_helper.rb

```diff
@@ -35,6 +37,18 @@ def can_destroy?(repository)
       RepositoryPolicy.new(current_user, repository).destroy?
   end
 
+  # Returns if any security module is enabled
```

Comment:
Any reason to move these functions up?

Reply:
It was giving me some errors on RSpec because they were not public 😕 (no idea why it was working before)